src/OMSimulatorLib/System.cpp

```diff
@@ -1689,6 +1692,9 @@ bool oms::System::copyResources()
 
 void oms::System::getAllResources(std::vector<std::string>& resources) const
 {
+  if (!values.empty() && !Flags::ExportParametersInline())
```

Comment:
this will not work, and values.empty() will always be false and the ssv files will not be pushed to resources

Reply:
the ssv file can be pushed directly in `exportoFile()` in `Model.cpp` which is easily understandable i guess

Reply:
Why will values.empty() always be false?